vcd/sample_vcd_test_config.json

```diff
@@ -29,6 +29,11 @@
     "//catalog": {
       "//name": "existingCatalogName",
       "//catalogItem": "existingOvaNameInCatalog"
+    },
+    "providerVdc": {
```

Comment:
This isn't provider VDC, it's **organization** VDC. Better not use the name "provider" together with "VDC", because we will add provider VDC support some time in the future.

Reply:
The provider Vdc information is needed when creating an Org Vdc.
I think it should stay as it is. Perhaps we can add a comment to explain why we need it in the configuration

Reply:
Yes, Giuseppe is right. This provider VDC information to create organization VDC in unit tests.